code/analyticsedge/src/phone/java/com/adobe/marketing/mobile/AnalyticsExtension.java

```diff
@@ -414,6 +433,31 @@ private void sendAnalyticsHit(final HashMap<String, String> analyticsVars, final
         MobileCore.dispatchEvent(event, null);
     }
 
+    /**
+     * Adds the `Analytic id` and `Visitor id` to Analytics vars in analytics hit.
+     * @param analyticsVars the analytics vars {@link Map}
+     */
+    private void addAIDAndVIDToAnalyticsVars(final Map<String, String> analyticsVars) {
+        if (platformServices != null) {
```

Comment:
Minor suggestion :- I feel it makes easier to read code if we exit early instead of nesting if statements.